src/components/Button/Button.theme.ts

```diff
@@ -0,0 +1,14 @@
+import { ComponentsTheme, parse } from '../ThemeProvider/Ant'
+import Theme from '../../themes/schema'
+
+/**
+ * Generates a Ant theme configuration for Button component based on a theme configuration.
+ *
+ * @link https://ant.design/components/button#design-token
+ *
+ * @param {Partial<Theme>} theme - theme configuration.
+ * @returns {Partial<ComponentsTheme>} The generated Button Ant theme configuration.
+ */
+export default ({ typography }: Partial<Theme>): ComponentsTheme['Button'] => ({
+  contentFontSizeLG: parse(typography?.bodyS?.fontSize),
```

Comment:
I would also connect contentFontSize to the action part of typography inside the theme. If we fill that part is not supporting buttons correctly, I would rather change it to have 3 different sizes. bodyS is not so correct for buttons.

Reply:
I agree, but currently `action` has `"fontSize": "16"`, while our buttons always have 14px. Should we change the value in the theme file?

Reply:
I would actually add a actionLg property to the theme for the contentFontSizeLG token, and use action for the contentFontSize token.